Sources/Model/Settings.swift

```diff
@@ -26,7 +26,7 @@ import Photos
 @objc(BSImagePickerSettings) // Fix for ObjC header name conflicting.
 @objcMembers public class Settings : NSObject {
     public static let shared = Settings()
-
+    
```

Comment:
code style: please try to change you indentation settings to match the default for the project so there no "empty" changes